src/views/ConfigurationActionsView.vue

```diff
@@ -13,9 +13,13 @@
             <li>HTTP Request actions can be used to call external APIs, like servers, vehicles, cameras, etc.</li>
             <li>MAVLink Message actions allow you to send specific MAVLink messages to vehicles.</li>
             <li>JavaScript actions give you full flexibility by allowing you to write custom code.</li>
+            <li>
+              You can also link actions to Data Lake variables, by clicking the link icon. When configuring that, any
+              changes to the variable value will automatically trigger the linked action.
+            </li>
```

Comment:
```suggestion
            <li>
              The link button can be used to link Actions to Data Lake variables, so that changes to the variable
              value will automatically trigger the linked action.
            </li>
            <li>Actions can also be tested/run manually, using the play button.</li>
```